src/Rest/RestApiBrowser.php

```diff
@@ -116,17 +116,19 @@ public function sendRequest($method, $url, $body = null)
     /**
      * @param string $method
      * @param string $uri With or without host
-     * @param string|resource|array $body
+     * @param string|array $body
      */
     private function send($method, $uri, $body = null)
     {
         if (!$this->hasHost($uri)) {
             $uri = rtrim($this->httpClient->getConfiguration()->getBaseUri(), '/') . '/' . ltrim($uri, '/');
         }
+        $body = is_array($body) ? http_build_query($body) : $body;
         $stream = new Stream('php://memory', 'rw');
-        if ($body) {
+        if (is_string($body)) {
```

Comment:
Yup you're right :+1:  so we then cast it explicitely into string in the stream write then.